apps/api/src/domain/games/memory-match/queries/memory-match-queries.fetch-by-id.e2e.spec.ts

```diff
@@ -50,63 +59,86 @@ const publishedRound = buildTestInstance(MemoryMatchRound, {
     name: buildMultilingualTextWithSingleItem('published round'),
 });
 
+const unpublishedRound = buildTestInstance(MemoryMatchRound, {
+    id: buildDummyUuid(123),
+    isPublished: false,
+});
+
 /**
  * TODO[https://coscrad.atlassian.net/browse/CWEBJIRA-305]
  * Return unpublished rounds to admin users.
  */
 describe(`when querying for a memory match round: fetch by Id`, () => {
-    const testDatabaseName = generateDatabaseNameForTestSuite();
-
     let app: INestApplication;
 
-    let databaseProvider: ArangoDatabaseProvider;
-
     let memoryMatchRepository: IMemoryMatchRepository;
 
-    beforeEach(async () => {
-        // TODO use a constant for the collection name
-        await databaseProvider.getDatabaseForCollection('memory_match_rounds').clear();
```

Comment:
Where \ how do you do this now?

Reply:
Good question!!

Reply:
I tried to fix this but am now getting the error:

`TypeError: Cannot read properties of undefined (reading 'getDatabaseForCollection')`